src/controllers/components/item.js

```diff
@@ -1,9 +1,20 @@
-import { searchItem, deleteItem } from '../../lib/api-client';
+import { searchItem, getAllItems, deleteItem, addFault } from '../../lib/api-client';
 import { Dispatcher } from 'consus-core/flux';
 
 export default class ItemController {
 
-    static deleteItem(item) {
+    static addFault(faultObj){
+        return addFault(faultObj.itemAddress, faultObj.fault).then( res => {
```

Comment:
Nit: should name this `apiAddFault` or the like, so two methods don't have the same name.